libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/utils/ActiveLearningUtils.java

```diff
@@ -1,2 +1,104 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
+package com.microsoft.bot.ai.qna.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.microsoft.bot.ai.qna.models.QueryResult;
```

Comment:
Shouldn't this import go on top of the others?

Reply:
Done